arch/arm64/boot/dts/qcom/sdm632-fairphone-fp3.dts

```diff
@@ -70,6 +70,15 @@
 	status = "okay";
 
 	/* Himax touchscreen @ 0x48 */
+	touchscreen@48 {
+		compatible = "himax,hx83112b";
+		reg = <0x48>;
+		interrupt-parent = <&tlmm>;
+		interrupts = <65 0x2008>;
```

Comment:
instead of 0x2008 use proper flags, so that should be `IRQ_TYPE_LEVEL_LOW` , not sure what the remaining `0x2000` should be, that might be `IRQ_NO_BALANCING` but please try if just LEVEL_LOW works

Reply:
It works with just `IRQ_TYPE_LEVEL_LOW`.